docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,153 @@
+
+ .. Click the *Server path* tab.
+ .. Enter the *Labels path* of the server-side file or directory containing the label files in the field.
+ .. Select *Scan recursively* if relevant.
+ .. Click *Save*.
+
+. Click *Next*.
++
+[NOTE]
+====
+* To enable or disable a label, toggle the *Enable* switch of the label.
+* To delete label, click the Options menu {kebab} beside the label and select *Delete*.
+====
++
+. Click *Next*.
++
+You can customize your analysis by setting some or all of the options on the *Advanced options* page.
++
+. On the left side of the *Advanced options* page, complete the following fields:
+
+* *Target*: Select from the list. By default, these are the targets you chose previously.
+* *Source*: Select from the list.
+* *Exclude tags*: Rules with these tags will not be processed. Select from the list.
+* *Additional classpath*: Enter a space-delimited list of additional `.jar` files or directories to add to the class path so that they are available for decompilation or other analysis.
+* *Application name*: Full name of an application.
+* *Mavenize group ID*: Enter the Mavenize group ID.
+* *Ignore path*: Enter a location, in addition to `${user.home}/.mta/ignore/`, for {ProductShortName} to identify files that should be ignored.
+
+. On the right side of the *Advanced options* page, set the following toggle switches. By default, all are set to disabled.
+
+* *Export CSV*: Exports the report data to a CSV file in your local file system.
+* *Disable Tattletale*: Disables generation of a Tattletale report for each application.
+* *Class Not Found analysis*: Enables analysis of Java files that are not available on the class path. This option should not be used if some classes will be unavailable at analysis time.
+* *Compatible Files report*: Enables generation of the Compatible Files report. This report may take a long time for large applications.
+* *Exploded app*: Indicates that the provided input directory contains source files for a single application.
+* *Keep work dirs*: Instructs {ProductShortName} to not delete temporary working files, such as the graph database and extracted archive files. This option is useful for debugging purposes.
+* *Skip reports*: If enabled, HTML reports are not generated. Enable this toggle if you enabled *Export CSV*.
+* *Allow network access*: Allows network access for features that require it. Currently only validating XML schemas against external resources relies on Internet access. Note that using this option comes with a performance penalty.
+* *Mavenize*: Creates a Maven project directory structure based on the structure and content of the application.
+* *Source mode*: Indicates that the application to be evaluated contains source files rather than compiled binaries.
+
+. Click *Next*.
++
+In the *Review* screen, you can review your analysis and then decide to do one of the following: save your analysis without running it now, save  and run it now, or modify it before saving it.
```

Comment:
I would shorten this and the next few steps to this step:

Review your analysis [ANALYSIS or PROJECT????] and click Save or Save and Run, to run the analysis immediately.
+
Your project appears in the project list.